### PR TITLE
Add "My profile" link to website footer

### PR DESCRIPTION
**Pull request by DatGuy1**
Pull request checklist:

- [ ] `CHANGELOG.md` was updated, if applicable
- [ ] Documentation in docs/ or install-docs/ was updated, if applicable

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -29,6 +29,11 @@
             {% endif %}
             {{ menuitem.caption }}</a>
         {% endfor %}
+        {% if session.user %}
```
Comment (RAnders00) [inline templates/layout.html:32]:
```suggestion
        {% if session.user is not none %}
```

File: templates/layout.html [hunk 2]
```diff
@@ -29,6 +29,11 @@
...
+        <a class="item{% if active_page == "me" %} active {% endif %}" href="/user/{{ session.user.name  }}">
```
Comment (RAnders00) [inline templates/layout.html:33]:
```suggestion
        <a class="item{% if active_page == "me" %} active {% endif %}" href="/user/{{ session.user.login }}">
```

File: templates/layout.html [hunk 3]
```diff
@@ -29,6 +29,11 @@
...
+            Me
```
Comment (RAnders00) [inline templates/layout.html:34]:
```suggestion
            My profile
```

**Comment by pajlada** [comment]
@DatGuy1 please check out the changes we've made to see if you're still happy with this

**Comment by DatGuy1** [comment]
I personally prefer having it at the navbar, but this works if both of you seem to prefer it.

**Pull request merged**